src/main/java/salesforce/ui/pages/ProductPage.java

```diff
@@ -0,0 +1,18 @@
+package salesforce.ui.pages;
+
+import org.openqa.selenium.WebDriver;
+
+/**
+ * This class returns an instance of ProductPage.
+ */
+public class ProductPage extends BasePage {
+
+    public ProductPage(final WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    protected void waitForPageLoaded() {
+
```

Comment:
Could override this method.

Reply:
I will implement this method in the next PR